### PR TITLE
implement FB_render_model

### PR DESCRIPTION
**Pull request by awtterpip**
Implements the [`FB_render_model`](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XR_FB_render_model) extension.

### CODE REVIEW

File: openxr/src/session.rs [hunk 1]
```diff
@@ -308,6 +310,102 @@ impl<G> Session<G> {
...
+                self.as_raw(),
+                &RenderModelLoadInfoFB {
+                    ty: RenderModelLoadInfoFB::TYPE,
+                    next: ptr::null_mut(),
+                    model_key,
+                },
+                &mut buffer,
+            ))?;
+            let mut out = Vec::with_capacity(buffer.buffer_count_output as usize);
+            loop {
```
Comment (Ralith) [inline openxr/src/session.rs:380]: Could this use `get_arr`?

**Review by Ralith — CHANGES_REQUESTED** [review]
Thanks!

File: openxr/src/session.rs (hunk 1 above)
Comment (awtterpip) [reply openxr/src/session.rs]: check it now, i switched to using get_arr

**Review by Ralith — APPROVED** [review]
Thanks!

**Pull request merged**